public/friendlycode/js/fc/ui/bramble-proxy.js

```diff
@@ -215,11 +218,21 @@ define(["backbone-events", "fc/prefs"], function(BackboneEvents, Preferences) {
   // risk breaking Thimble's functionality,
   // but with a proper code audit they'll no
   // longer be needed
-  function empty() {}
+  function empty() {};
```

Comment:
elaborate? I did make a PR in the past but never told you about it. This is the PR in thimble that makes what I changed on the brackets side work